lib/interactive_viewer.ml

```diff
@@ -167,12 +231,17 @@ let view (patches : Patch.t list) =
                  | `ASCII 'h', [] ->
                      Lwd.set help true;
                      `Handled
+                 | `ASCII 't', [] ->
+                    toggle_view_mode ();
+                    `Handled
                  | _ -> `Unhandled)
                (W.string
                   "Type 'h' to go to the help panel, 'q' to quit, 'n' to go to \
-                   the next operation, 'p' to go to the previous operation");
+                   the next operation, 'p' to go to the previous operation. Press 't' to toggle view mode.");
         ]
   in
   W.vbox [ ui ]
 
 let start patch = Ui_loop.run ~quit ~tick_period:0.2 (view patch)
+
+
```

Comment:
You need to format with `dune build @fmt --auto-promote`.

Reply:
Still not formatted. If you use vscode, I suggest turning on formatting on save.